src/renderer/store/TagManagerStore.tsx

```diff
@@ -21,15 +24,75 @@ export const useTagManagerStore = (): UseTagManagerStore => {
     const [markedToKeep, setMarkedToKeep] = useAtom(markedToKeepAtom);
     const [hoveredId, setHoveredId] = useAtom(hoveredIdAtom);
 
+    // DELETE LOGIC
```

Comment:
Should it be considered?

Reply:
oh, my bad it's not a "Delete this logic" comment, but "This is the logic of "delete" feature".  
Maybe another comment with more sense should be used.